k8s/crd-samples/replica-crd.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cstorreplicas.openebs.io
+  namespace: default
+spec:
+  serviceAccountName: openebs-maya-operator
+  group: openebs.io
+  names:
+    kind: CstorReplica
```

Comment:
Change all occurances of CstorReplica to CStorVolumeReplica and make the changes according to the case

Reply:
ok